com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -1168,116 +1248,91 @@ private void HandleRawTransportPoll(NetworkEvent networkEvent, ulong clientId, A
             }
         }
 
-        private readonly NetworkBuffer m_InputBufferWrapper = new NetworkBuffer(new byte[0]);
-        private readonly MessageBatcher m_MessageBatcher = new MessageBatcher();
-
-        internal void HandleIncomingData(ulong clientId, ArraySegment<byte> payload, float receiveTime)
+        public unsafe int SendMessage<TMessageType, TClientIdListType>(in TMessageType message, NetworkDelivery delivery, in TClientIdListType clientIds, bool serverCanSendToServerId = false)
```

Comment:
I was going to say "let's overload this method with alternatives" and you already did that, nice!